src/components/Tabs/Tabs.module.css

```diff
@@ -20,33 +20,6 @@
   padding-bottom: var(--eds-size-2);
 }
 
-/**
- * Fade scrollable indicators to display if there is scrollable area on either side.
- *
- * The color "white" is arbitrary and any non transparent color can be used here.
- */
-.tabs--scrollable-left {
-  -webkit-mask-image: -webkit-linear-gradient(
-    left,
-    transparent,
-    white var(--eds-size-8)
-  );
-}
-
-.tabs--scrollable-right {
-  -webkit-mask-image: -webkit-linear-gradient(right, transparent);
-}
-
-.tabs--scrollable-left.tabs--scrollable-right {
-  -webkit-mask-image: -webkit-linear-gradient(
-    left,
-    transparent,
-    white var(--eds-size-8),
-    white calc(100% - var(--eds-size-8)),
-    transparent 100%
-  );
-}
```

Comment:
Removed this scroll styling, because

1. It uses non-standard CSS
2. Seems like it has low value

Here's a pic of the styling (that's been deleted):

<img width="154" alt="Screen Shot 2022-09-29 at 1 40 47 PM" src="https://user-images.githubusercontent.com/2503289/193106843-454fd76d-d18a-46a7-8e87-cee6506517f9.png">

It's the sorta-transparency on the left.

Removing this cleans up a lot of code from Tabs.

Reply:
>2. Seems like it's low value

The purpose of this transparency fade is to communicate to the user that the tabs extend beyond the edge of the page. I think that's unexpected and many users would be confused by that. The fade was design's solution to communicating that non-standard UX to the user. I don't think we should remove it unless we have an alternative solution.

Reply:
Heh, I sort of assumed this came from BM's template. But if we designed it this way, I'll add it back.

I do still question the non-standard CSS (there's probably other ways of doing it). There's also the browser's scrollbar, which indicates that there are more things (and can be scrolled).

But anyway, will add it back. Bummer it takes JS to do 🤔

Reply:
Oh yeah, that's true. The scrollbars only show up when you're already scrolling.

I think I just don't like this solution because it seems like a lot of trouble to show a bit of transparency (will people even notice it?). But that doesn't mean it's wrong.

Reply:
>will people even notice it?

The designer originally also had a right/left caret button on top of the transparent area, but honestly that just looked like a hassle to build so we decided to simplify a little and just do the transparency.